ui/src/styles/style.css

```diff
@@ -189,6 +192,10 @@ tr:hover td {
   color: red;
 }
 
+.managerNotice {
+  margin-bottom: 20px;
```

Comment:
Is this value of 20px related in anyway to the top margin in `.button-vote-wrapper` or the vertical margin in `.page h2 + *`? If they are supposed to be all equal, then maybe it is a good time to introduce a variable?